consensus/src/round_manager.rs

```diff
@@ -1007,22 +1107,73 @@ impl RoundManager {
                         BlockStage::QC_AGGREGATED,
                     );
                 }
-                self.new_qc_aggregated(qc, vote.author()).await
+                QC_AGGREGATED_FROM_VOTES.inc();
+                let result = self.new_qc_aggregated(qc.clone(), vote.author()).await;
+                if self.onchain_config.order_vote_enabled() {
+                    if result.is_ok() {
+                        let _ = self.broadcast_order_vote(vote, qc.clone()).await;
```

Comment:
I think we can move the vote above the "new_qc_aggregated" to avoid blocking by block fetch